lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2676,6 +2697,26 @@ public function get_styles_block_nodes() {
 		return static::get_block_nodes( $this->theme_json );
 	}
 
+	/**
+	 * A public helper to get all styles nodes path in theme.json,
+	 * including elements and block styles variations. This is useful
+	 * when iterating over all style nodes to search for or replace any values.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @return array An array of paths to style nodes in theme.json.
+	 */
+	public function get_styles_nodes_paths() {
```

Comment:
Needs tests

Reply:
Nit: The naming here with three successive plurals seems a bit clunky. What about `get_style_node_paths`?

This avoids each word within the name "modifying" those before it making it a little less awkward and easier to parse.